mods/ra2/rules/soviet-structures.yaml

```diff
@@ -32,6 +32,21 @@ nacnst:
 		Facing: 96
 		DeployCursor: undeploy
 		RequiresCondition: factundeploy && !build-incomplete
+	TransformsIntoMobile:
+		RequiresCondition: factundeploy
+		Locomotor: tracked
+		Voice: Move
+		RequiresForceMove: true
+	TransformsIntoRepairable:
+		RequiresCondition: factundeploy
+		RepairActors: nadept, gadept, caoutp
```

Comment:
This is till wrong.

Reply:
`nadept` is the soviet counter-part. Why is this wrong for the soviet MCV?

Reply:
All units can get repaired from all Service depots, i don't see any reason why Soviet Service Depot should be first for Soviet MCV.

Reply:
Because that is most likely in a safe place at the mother base.

Reply:
I think changing the order would be an unnecessary inconsistency in the code. The order changes nothing ingame.

Reply:
Why are you making me change this all the time if it doesn't do anything? I think my way of setup is more consistent with the faction specific buildings.